OpenFEC-Data-Visualization/blog.md

```diff
@@ -0,0 +1,285 @@
+            labels: names,
+            /* create an array of different data values for each name, each having respective labels, colors,
+               and data that we obtained from the API response earlier*/
+            datasets: [
+                {
+                label: 'Operating Expenditures',
+                data: operating_expenditures,
+                backgroundColor: "rgba(106, 220, 123, 1)",
+                borderColor: "black",
+                borderWidth: 1
+            		},
+            		{
+                label: "Net Receipts",
+                data: net_receipts,
+                backgroundColor: "rgba(71, 175, 255, 1)",
+                borderColor: "black",   
+                borderWidth: 1,                  
+                },
+                {
+                label: "Fundraising Disbursements",
+                data: net_receipts,
+                backgroundColor: "rgba(136, 0, 240, 1)",
+                borderColor: "black",    
+                borderWidth: 1,                  
+                },
+                {
+                label: "Individual Contributions Less Refunds",
+                data: individual_contributions_less_refunds,
+                backgroundColor: "rgba(233, 114, 114, 1)",
+                borderColor: "black",    
+                borderWidth: 1,                  
+                }
+            ]
+        },
+```
+
+Once you’re done labeling and customizing with each data set, you can add options to create a title for the graph.
+
+```javascript
+// tinker with various options for the chart
+        options: {
+            // set a title
+            title: {
+                display: true,
+                text: 'OpenFEC Data Visualization'
+            }
+        }
+    });
+    
+    }
+}
+```
+
+That's it! You've completed the coding!
+
+## Expected Output
+
+Let's see how the graph looks! Save your `OpenFEC.html` and `OpenFEC.js` files. Look for `OpenFEC.html` in your file directory and right click it. Open the file with a web browser and it should look like below!
+
+![img](https://lh4.googleusercontent.com/vrlUSf7GTKdVCinKZix2Lo8O5Mg0WE70j79mNBWtX7td2tRCp6sGvg2OJQnJO2fg34-enEqBEtlg9rRIF9YNSfhrh5O07Z2odv_oUvuyBjHozHjjfGHPIo0F4NOTUm0OkN6rQtWW)
+
+## Conclusion
+
+From this tutorial, you should have learned
+
+* How to use the OpenFEC API to obtain desired information
+* What JSON is
+* How to make an API request from your JavaScript code using `XMLHttpRequest` and how to extract information from the response
+* How to visualize data using ChartJS in order to make a bar chart
+
+While this example was simple, it served to illustrate the power of understanding how to use APIs to extract information and how to use this information to create your own charts. The variety of information you can explore even within one API and how you can visualize the information for trend analysis is an extremely powerful tool that helps in discovering a multitude of information.
```

Comment:
I do think the conclusion can be a bit more specific in terms of its implications for informed voting, just to summarize the article and tie it back to the introduction. Otherwise, this looks great!